java/src/main/java/ai/rapids/cudf/ParquetWriterOptions.java

```diff
@@ -56,6 +61,14 @@ public Builder withTimestampInt96(boolean int96) {
       return this;
     }
 
+    /**
+     * Flattened precision values for all the decimal columns
+     */
+    public Builder withPrecisionValues(int... precisionValues) {
+      Arrays.stream(precisionValues).forEach(i -> precisionList.add(i));
```

Comment:
Do we really want to support calling this method iteratively?  I could see users being confused, thinking this will override previous calls.

I think we either want this to override any previous call or take a single precision value and append it.  Personally I prefer the former.  That would also simplify the builder, as it can track the array directly.

Either way the Javadoc should be clear on the semantics of how the input will be handled on subsequent invocations.